ipfsconn/ipfshttp/ipfshttp_test.go

```diff
@@ -99,6 +99,33 @@ func TestIPFSPin(t *testing.T) {
 	}
 }
 
+func TestIPFSPinRefs(t *testing.T) {
```

Comment:
Not so important as this is a test but we could abstract away the functionality of TestIPFSPin and TestIPFSPinRefs into a function that takes in a an ipfsConnector object so that `ipfs.config.PinMethod` can be set to "refs" or left unset before the call and get rid of most of this duplicate code.